conans/test/integration/cache/download_cache_test.py

```diff
@@ -126,7 +149,7 @@ def source(self):
         self.assertIn("ConanException: md5 signature failed for", client.out)
         self.assertIn("Provided signature: kk", client.out)
         self.assertIn("Computed signature: 9893532233caff98cd083a116b013c0b", client.out)
-        self.assertEqual(1, len(os.listdir(cache_folder)))  # Nothing was cached
+        self.assertEqual(2, len(os.listdir(cache_folder)))  # Nothing was cached
```

Comment:
Why is this magic number 2? Can we add comment like line 173?

Reply:
Done, please re-check